packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -53,33 +53,31 @@ export default class PhoneNumberInput extends React.Component<Props, State> {
 
   componentDidMount() {
     if (this.props.defaultCountry) {
-      this.onChangeCountryQuery(this.props.defaultCountry)
+      this.ChangeCountryQuery(this.props.defaultCountry)
     }
 
     if (this.props.defaultCountryCode) {
       const country = this.state.countries.getCountryByPhoneCountryCode(
         this.props.defaultCountryCode
       )
 
-      this.onChangeCountryQuery(country.displayName)
+      this.ChangeCountryQuery(country.displayName)
     }
 
     if (this.props.defaultPhoneNumber) {
       this.onChangePhoneNumber(this.props.defaultPhoneNumber)
     }
   }
 
-  async getPhoneNumberFromNativePicker() {
-    // linter doesn't allow following code without splitting this code in two lines
-    const number = await SmsRetriever.requestPhoneNumber()
-    return number
+  async _getPhoneNumberFromNativePickerAndroid() {
```

Comment:
Haven't seen the _ used in other parts of our codebase, ok to just keep as `getPhoneNumberFromNativePickerAndroid`?